File: authentication/authentication/src/main/java/com/example/authentication/model/Mobile.java
package com.example.authentication.model;

import com.example.authentication.entity.RatingEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Mobile {
    private Long mobileID;
    private String mobileName;
    private String mobileModel;
    private String mobileType;
    private String mobileDescription;
    private Integer mobileQuantity;
    private Double mobilePrice;
    private String imageUrl;
    private RatingEntity rate;
    private LocalDateTime createAt;
    private LocalDateTime updateAt;
}

File: Joomia/app/src/main/java/com/kanyideveloper/joomia/feature_products/domain/model/Mobile.kt
package com.kanyideveloper.joomia.feature_products.domain.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Mobile(
    val mobileID: Int,
    val mobileType: String,
    val mobileDescription: String,
    val imageUrl: String,
    val mobilePrice: Double,
    val mobileQuantity: Int,
    val rating: Rating,
    val mobileName: String,
    val mobileModel: String
) : Parcelable
